wagtail/admin/templates/wagtailadmin/shared/panel.html

```diff
@@ -54,7 +54,7 @@ <h2 class="w-panel__heading {% if heading_size == "label" %}w-panel__heading--la
         </div>
     {% endif %}
 
-    <div id="{{ content_id }}" class="w-panel__content">
+    <div id="toggle-target {{ content_id }}" class="w-panel__content " data-w-panel-target="item">
```

Comment:
```suggestion
    <div id="{{ content_id }}" class="w-panel__content " data-w-panel-target="item">
```

As per the original issue, let's use `container` or maybe `content` for the content that shows or hides.

Reply:
in this should i change item -> content ?

Reply:
Yeah. The original issue has this target name specifically called out to be `container` but `content` also makes sense. 

Item is a tad too generic I think